src/lib/identifiers.js

```diff
@@ -81,4 +83,31 @@ if (s) {
   return RAW_READ_REGEX.test(s)
 }
 return false
+}
+
+export async function validateTaxonomy(s) {
```

Comment:
not sure this file is the correct place for this. this file otherwise contains only constants and functions that use them so I'm not sure making API calls in here is a good idea. is this function used anywhere other than scoreseteditor? can the code live there?

Reply:
I didn't notice that only ScoreSetEditor needs this function. I referred to other functions such as `validateDoi` but they're used in both ExperimentEditor and ScoreSetEditor pages. I'll move it to ScoreSetEditor page.